packages/design-system/src/components/infiniteScroller/index.js

```diff
@@ -176,7 +180,7 @@ const InfiniteScroller = ({
 
     return (
       <LoadingContainer>
-        <LoadingSpinner animationSize={50} circleSize={6} />
+        <LoadingSpinner {...loadingSpinnerProps} />
       </LoadingContainer>
     );
   }, [allDataLoadedMessage, canLoadMore, loadState]);
```

Comment:
```suggestion
  }, [allDataLoadedMessage, canLoadMore, loadState, loadingSpinnerProps]);
```

Though maybe need to run Prettier on that

Reply:
Fixed.